### ISSUE TITLE
[Popup] - Can't set inverted by Popup Settings

### ISSUE THREAD

**Issue by wuchong** [Semantic-Org/Semantic-UI#3367]
**Steps to Reproduce**
1. Set a refresh icon 

  ```html
  <i class="refresh_tag refresh icon link" ></i>
  ```

2. use js to popup content with inverted configuration

  ```javascript
  $('.refresh_tag.refresh').popup({
              content: "refresh",
              inverted: true
  });
  ```

**Expected**
The popup is  inverted

**Result**
The Popup is not inverted

**Testcase**
http://jsfiddle.net/imjark/b77n89tg/1/

**Comment by jlukic** [comment]
Apologies, the docs are wrong. There is not a setting 'inverted' in popup.

Instead you can specify variations using `variation` setting
http://jsfiddle.net/5pw7ppfv/

**Comment by jlukic** [comment]
I've fixed docs with `2.1.7`
